ci/docker/forge.Dockerfile

```diff
@@ -40,7 +40,7 @@ apt-get install -y \
 
 # As a convention, we pin all python packages to a specific version. This
 # is to to make sure we can control version upgrades through code changes.
-python -m pip install pip==23.2.1
+python -m pip install pip==23.2.1 cffi==1.16.0
```

Comment:
cffi which is a dependency of ray-release requires a _cffi_backend interface on the host machine; this force the installation of that interface (https://buildkite.com/ray-project/premerge/builds/16041#018cecda-91b8-4757-8dfe-68a15b10692c/181-254)